.storybook/main.js

```diff
@@ -31,6 +35,7 @@ module.exports = {
     "@storybook/addon-google-analytics",
     "@storybook/addon-links",
     "@storybook/addon-toolbars",
+    "@storybook/addon-mdx-gfm",
```

Comment:
Again, this is very likely just an oversight. Storybook automatically add it when you run their upgrade wizard to 7 to assist in the update. I don't think it should be needed for us now as I think I've updated all MDX files accordingly but I will check to make sure before removing it otherwise I'll have to raise a ticket to ensure we complete that conversion